FindMe/Location/AlertManager.m

```diff
@@ -0,0 +1,44 @@
+//
+//  AlertManager.m
+//  FindMe
+//
+//  Created by Nicholas Wayoe on 7/27/20.
+//  Copyright © 2020 Nicholas Wayoe. All rights reserved.
+//
+
+#import "AlertManager.h"
+#import "Mailgun/Mailgun.h"
+#import "DatabaseManager.h"
+#import "Contact.h"
+
+@implementation AlertManager
+
++ (void)makeStringFromPlacemarkAndSendEmail:(CLPlacemark *)decodedLocation withContact:(Contact *)contact
+{
+    NSString *message = [NSString stringWithFormat: @" \
+                         Address : %@ %@ %@ \
+                         \
+                         city : %@ \
+                         \
+                         Neighbourhood: %@ \
+                         \
+                         State : %@ \
+                         \
+                         Country : %@ \
+                         \
+                         Your are receiving this notification because put you as emergency Contact. \
+                         ",decodedLocation.subThoroughfare, decodedLocation.thoroughfare, decodedLocation.postalCode, decodedLocation.locality, decodedLocation.subLocality, decodedLocation.administrativeArea, decodedLocation.country];
```

Comment:
Can you use any string appending methods to make this cleaner?

Reply:
Also this assumes that all of those fields exist. Will they always exist?

Reply:
I think so. From the document, it has all those fields.